examples/eliza/tutorial_eliza.ipynb

```diff
@@ -0,0 +1,161 @@
+    "```python\n",
+    "pipeline = Pipeline[DataPack]()\n",
+    "pipeline.set_reader(RawDataDeserializeReader())\n",
+    "pipeline.add(ElizaProcessor())\n",
+    "pipeline.serve()\n",
+    "```\n",
+    "Here we set [`RawDataDeserializeReader`](https://github.com/asyml/forte/blob/master/forte/data/readers/deserialize_reader.py#L90) as a reader of the pipeline since we expect the input request will be a sequence of serialized `DataPack` strings. This reader is able to deserialize these strings to `DataPack`s.\n",
+    "\n",
+    "[`ElizaProcessor`](https://github.com/asyml/forte/blob/master/forte/processors/nlp/eliza_processor.py) is responsible for generating responses based on Eliza rules from the input queries. The responses will be appended to the text payload of input `DataPack` annotated as `Utterance`.\n",
+    "\n",
+    "[`Pipeline.serve(host, port)`](https://github.com/asyml/forte/blob/master/forte/pipeline.py#L602) will start a pipeline service at a specific endpoint. You may configure `host` and `port` to specify the endpoint.\n",
+    "\n",
+    "### Call a Pipeline Service\n",
+    "After setting up the service, you will be able to access it from `http://{host}:{port}`. You can also call it from another forte pipeline with [`RemoteProcessor`](https://github.com/asyml/forte/blob/master/forte/processors/misc/remote_processor.py). In the preloaded `Eliza` project, `stave` has already set up a forte pipeline to invoke the Eliza service:\n",
+    "```python\n",
+    "# Adapted from https://github.com/asyml/stave/blob/master/simple-backend/stave_backend/handlers/nlp.py#L49\n",
+    "pipeline = Pipeline[DataPack](do_init_type_check)\n",
+    "pipeline.set_reader(RawDataDeserializeReader())\n",
+    "pipeline.add(RemoteProcessor(), config)\n",
+    "pipeline.initialize()\n",
+    "``` \n",
+    "[`RawDataDeserializeReader`](https://github.com/asyml/forte/blob/master/forte/data/readers/deserialize_reader.py#L90) is set as the reader of this pipeline because `stave` stores `DataPack`s as serialized strings in database. Input queries entered by users are integrated into an existing chatbot `DataPack` and saved to database, which will be fed into the pipeline above.\n",
+    "\n",
+    "[`RemoteProcessor`](https://github.com/asyml/forte/blob/master/forte/processors/misc/remote_processor.py) provides a wrapping of interactions with remote forte service endpoint. Each input DataPack from the upstream component will be serialized and packed into a POST request to be sent to a remote service, which should return a response that can be parsed into a DataPack to update the input. In the Eliza example, it will prepare a POST request from the deserialized `DataPack` (which contains the user inputs), and send it to the remote service we just set up, and then parse the response (which contains the generated text from `ElizaProcessor`) into a new `DataPack` that can be passed to the downstream components."
```

Comment:
Could you provide a bit introduction on how Stave renders DataPack, and how DataPack helps the comunication?